draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -160,12 +160,12 @@ The calculation of security-related fields such as Response-Authenticator, Messa
 
 RADIUS/(D)TLS does not use separate ports for authentication, accounting and dynamic authorization changes.
 The source port is arbitrary.
-For considerations regarding the multi-purpose use of one port for authentication and accounting see {{radius_datagrams}}.
+For considerations regarding the multi-purpose use of one port for authentication and accounting see {{radius_packets}}.
 
 RADIUS/TLS servers MUST immediately start the TLS negotiation when a new connection to the RADIUS/TLS port is opened.
 They MUST close the connection and discard any data sent if the connecting client does not start a TLS negotiation or if the TLS negotiation fails at any point.
 
-RADIUS/DTLS servers MUST silently discard any packet they receive over the RADIUS/DTLS port that is not a new DTLS negotiation or a packet sent over a DTLS session established earlier.
+RADIUS/DTLS servers MUST silently discard any UDP datagram they receive over the RADIUS/DTLS port that is not a DTLS handshake message or a DTLS record sent over a DTLS session established earlier.
```

Comment:
I think the change to "handshake or record" is dangerous.
What we mean is: either it's DTLS or it gets thrown away. with this change we would restrict what type of DTLS messages can be sent and that's not what we want.